src/main/java/core/basesyntax/dao/StorageDaoImpl.java

```diff
@@ -1,7 +1,7 @@
 package core.basesyntax.dao;
 
 import core.basesyntax.db.Storage;
-import core.basesyntax.model.User;
+import core.basesyntax.service.model.User;
 
 public class StorageDaoImpl implements StorageDao {
     private static Long index = 0L;
```

Comment:
Consider using an `AtomicLong` for the `index` field to ensure thread safety when incrementing the index in a concurrent environment. This is important if multiple threads will be interacting with `StorageDaoImpl`.